sourcemash/frontend/assets.py

```diff
@@ -39,8 +39,10 @@
                 "js/views/feed_view.js",
                 "js/views/saved_view.js",
                 "js/views/browse_view.js",
+                "js/views/register_view.js",
```

Comment:
No, the login form is strictly part of the sidenav. Register view is only nice b/c then it can be seen side-by-side with the login form in the sidenav.